draft-ietf-rtcweb-jsep.xml

```diff
@@ -405,6 +405,29 @@ setRemote(ANSWER) |                                   |
         APIs to allow the application to control various session parameters,
         which will provide the necessary information to the browser about what
         sort of SessionDescription to produce.</t>
+
+        <section title="CNAME Semantics" anchor="sec.cname">
```

Comment:
I think this section should move to a slightly different place - suggest the parent section, right before "Interactions With Forking". Doesn't really seem right under "Session Description Format".